.circleci/config.yml

```diff
@@ -1,46 +1,66 @@
 version: 2.1
 
-job-template: &job-template
-    working_directory: ~/circleci
-    steps:
-      - checkout
-      - restore_cache:
-          keys:
-            - composer-v1-{{ checksum "composer.json" }}
-            - composer-v1- 
-      - run: composer install -n && composer test
-      - save_cache:
-          key: composer-v1-{{ checksum "composer.json" }}
-          paths:
-            - vendor
-      - restore_cache:
-          keys:
-            - node-v1-{{ checksum "package.json" }}
-            - node-v1-
-      - run: yarn install && yarn run test && yarn run build:production
-      - save_cache:
-          key: node-v1-{{ checksum "package.json" }}
-          paths:
-            - node_modules
-
-jobs:
-  build-php73:
-    docker:
-      - image: circleci/php:7.3-stretch-node-browsers
-    <<: *job-template
-  build-php72:
-    docker:
-      - image: circleci/php:7.2-stretch-node-browsers
-    <<: *job-template
-  build-php71:
-    docker:
-      - image: circleci/php:7.1-stretch-node-browsers
-    <<: *job-template
+orbs:
+  sage:
+    executors:
+      node-10:
+        docker:
+          - image: 'circleci/node:10-stretch'
+      php-73:
+        docker:
+          - image: 'circleci/php:7.3-stretch'
+      php-72:
+        docker:
+          - image: 'circleci/php:7.2-stretch'
+      php-71:
+        docker:
+          - image: 'circleci/php:7.1-stretch'
+    jobs:
+      build-php:
+        parameters:
+          executor:
+            type: executor
+        executor: << parameters.executor >>
+        steps:
+          - run: php -v
+          - checkout
+          - restore_cache:
+              keys:
+                - composer-v1-{{ checksum "composer.lock" }}
+                - composer-v1-
```

Comment:
this meant to be here?

Reply:
Yes, it tells circleci to "fallback to using the latest cache if no exact match is found (See https://circleci.com/docs/2.0/caching/)". Doc: https://circleci.com/docs/2.0/language-php/#config-walkthrough